en/advanced/best-practice-security.md

```diff
@@ -152,6 +153,16 @@ app.use(session({
 }))
 ```
 
+## Prevent brute-force attacks against authorisation
+
+Make sure login endpoints are protected to make private data more secure.
+
+One simple and in the same time quite powerful way is to block authorisation attempts by 2 metrics:
```

Comment:
A simple and powerful technique is to block authorization attempts using two metrics: